tron/event.py

```diff
@@ -49,8 +49,8 @@ def __hash__(self):
 
 
 LEVEL_INFO = EventLevel(0, "INFO")  # Troubleshooting information
-LEVEL_OK = EventLevel(1, "OK")  # Expected behaviour
-LEVEL_NOTICE = EventLevel(2, "NOTICE")  # Troubling behaviour
+LEVEL_OK = EventLevel(1, "OK")  # Expected behavior
+LEVEL_NOTICE = EventLevel(2, "NOTICE")  # Troubling behavior
```

Comment:
https://www.dictionary.com/browse/behaviour

Reply:
Yea but, it is safer if we agree to only one version of spelling, namely US/American.
Otherwise we get (have had) incidents where someone spells `datacentre`.